src/triage/component/results_schema/schema.py

```diff
@@ -136,6 +151,21 @@ def delete(self, session):
         session.delete(self)
 
 
+class ExperimentModel(Base):
+    __tablename__ = "experiment_models"
+    __table_args__ = {"schema": "model_metadata"}
+
+    experiment_hash = Column(
+        String,
+        ForeignKey("model_metadata.experiments.experiment_hash"),
+        primary_key=True
+    )
+    model_hash = Column(String, primary_key=True)
```

Comment:
just curious, how come this can't be defined as a foreign key, here, as with `experiment_hash`?

Reply:
I'd like to be able to insert the hashes in the association table before we have all the details necessary to create the main row, and whether or not the experiment fails to create them (due to memory issues, or whatever).

Reply:
hmm ok